main.go

```diff
@@ -89,42 +99,104 @@ func main() {
 				},
 			},
 			{
-				Name:    "add git",
-				Aliases: []string{"git", "addgit"},
-				Description: `Allows canaveral to use your git credentials for repo management.
+				Name:    "add github credentials",
+				Aliases: []string{"gh", "github", "addgh", "addgithub"},
+				Description: `Allows canaveral to use your github credentials for repo management.
 					Username and password are required.
 					Username and password are stored in native storage for security.`,
-				Usage: "Add git info to canaveral.",
+				Usage: "Add github info to canaveral.",
+				Action: func(c *cli.Context) error {
+					if qFlag {
+						fmt.Println("(okay, I'll try to be quiet.)")
+					}
+					return gh.GHAddWrapper()
+				},
+			},
+			{
+				Name:        "remove github",
+				Aliases:     []string{"rgh", "remgh", "rgithub", "remgithub", "removegithub"},
+				Description: `Deletes your github credentials from native storage. Canaveral will no longer have any way to reference your githubcredentials.`,
+				Usage:       "Remove github info from canaveral",
+				Action: func(c *cli.Context) error {
+					if qFlag {
+						fmt.Println("(okay, I'll try to be quiet.)")
+					}
+					return gh.RemGHCredsHandler()
+				},
+			},
+			{
+				Name:        "print github",
```

Comment:
I would prefer if we didn't use the term `print` since its relatively technical. Could we use language more similar to `show` or `display`? You don't have to change all the function names and stuff, since we know whats going on. I'm suggesting just to change the command name and descriptions.

Reply:
So, I used print as it seems like it would be less ambiguous and more familiar to the target audience seeing as it's a tool for developers. Thoughts? I will change it if you disagree as I'm not too set on it.

Reply:
Alright, fair enough